src/java.management/share/native/libmanagement/VMManagementImpl.c

```diff
@@ -38,12 +38,11 @@ Java_sun_management_VMManagementImpl_getVersion0
     char buf[MAX_VERSION_LEN];
     jstring version_string = NULL;
 
-    unsigned int major = ((unsigned int) jmm_version & 0x0FFF0000) >> 16;
```

Comment:
Can you please revert this unrelated changes as well?

Reply:
Sure, this was a mistake I re added the unused line but at the wrong position in the file.